docker/zap-api-scan.py

```diff
@@ -241,8 +242,8 @@ def main(argv):
     if len(target) == 0:
         usage()
         sys.exit(3)
-    if format != 'openapi' and format != 'soap' and format != 'graphql':
-        logging.warning('Format must be either \'openapi\', \'soap\', or \'graphql\'')
+    if format not in SUPPORTED_FORMATS:
```

Comment:
here and log levels can be validated by switching from `getopt` -> `ArgumentParser`. In python2, Argument parser was not in the PSL, while in python3 you have it already and you don't need to install it anymore.